var/spack/repos/builtin/packages/py-netcdf4/nompi.patch

```diff
@@ -0,0 +1,11 @@
+--- a/setup.py
```

Comment:
This file can be made shorter:
```patch
--- a/setup.py
+++ b/setup.py
@@ -395 +395 @@ if 'sdist' not in sys.argv[1:] and 'clean' not in sys.argv[1:] and '--version' n
-    has_parallel_support = check_has_parallel_support(inc_dirs)
+    has_parallel_support = False
```

Reply:
The patch shown above does not work:
```
patching file setup.py
Hunk #1 FAILED at 395.
1 out of 1 hunk FAILED -- saving rejects to file setup.py.rej
```
I manually changed the one line and ran `diff -Naur` on the original and modified file, and I get exactly the patch that I submitted originally. I'll keep that version (but rename it as suggested below).

Reply:
I wonder what is the version of `patch` that cannot handle the patch file.